Source/ACE/Entity/WorldObject.cs

```diff
@@ -1963,7 +1991,7 @@ private WeenieHeaderFlag SetWeenieHeaderFlag()
             if (AmmoType != null)
                 weenieHeaderFlag |= WeenieHeaderFlag.AmmoType;
 
-            if (Value != null)
+            if (Value != null && (Value > 0))
```

Comment:
Is it correct behavior that it is impossible to create an item with Value 0, or does the client interpret no sent value as Value 0?